src/es/components/molecules/simpleForm/SimpleForm.js

```diff
@@ -22,7 +22,7 @@ import { Shadow } from '../../prototypes/Shadow.js'
  *
  * }
  */
-export default class SimpleForm extends Shadow() {
+export default class SimpleForm extends Validation() {
```

Comment:
I would handle this analog the [Wrapper](https://github.com/mits-gossau/web-components-toolbox/blob/master/src/es/components/organisms/wrapper/Wrapper.js#L21), except this extend has Zero-Risk to effect the SimpleForm.

First:
`export const SimpleForm = (ChosenHTMLElement = Shadow) => class SimpleForm extends ChosenHTMLElement {`

Second:
`export default class SimpleFormValidation extends SimpleForm(Validation())`

Then you have a new Class SimpleFormValidation and you can use both independent. What do you think?

Reply:
Fixed at Pair-programming together